Client/src/components/common/Sidebar/index.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import './index.css'
+import Linklst from './Linklst/index';
+const links = [[['Home'],['Course'], ['Student']]];
+
+export default function Sidebar() {
+  return (
+    <div className='sidebar'>
+      <img src='https://i.imgur.com/m6QdXqO.png' alt='AMDIEAST logo' className='img' width='200px' />
+      <h3 className='title' >AMIDEAST EL</h3>
```

Comment:
no style in the js files 
 width='200px'

Reply:
Done :)